example/conf/sweeper/wandb.yaml

```diff
@@ -0,0 +1,113 @@
+# @package _global_
+defaults:
+  - override /hydra/sweeper: wandb
+
+hydra:
+  sweeper:
+    # See https://docs.wandb.ai/guides/sweeps/configuration for types of configs
+    wandb_sweep_config:
+      # (Required) The name of the sweep, displayed in the W&B UI.
+      name: hydra_plugin_test
+
+      # (Required) Specify the search strategy.
+      method: bayes
+
+      # (Optional) Specify the metric to optimize (only used by certain search
+      # strategies and stopping criteria). Default is {}.
+      metric:
+        name: out
+        goal: minimize
+
+        # (Optional) Goal value for the metric you're optimizing. When any run
+        # in the sweep achieves that target value, the sweep's state will be set
+        # to finished. This means all agents with active runs will finish those
+        # jobs, but no new runs will be launched in the sweep.
+        # target: 0.0
+
+      # (Optional) Number of agents to launch in a batch until budget is
+      # reached. Use a multi-processing launcher to have these agents execute
+      # simultaneously, otherwise it's a useless option since they'll execute in
+      # sequence. Default is 1.
+      num_agents: 2
+
+      # (Optional) Total number of agents to launch (must be >= num_agents)
+      # Default is 1.
+      budget: 4
```

Comment:
Some of the parameters here looks pretty similar to what an user would have in a wandb `sweep.yaml` file. Would it be possible for you to group the `sweep.yaml` parameters together and wrap them within a comment block like:
```
######### Beginning sweep.yaml ############
method:
metric:
. . .
######## End sweep.yaml ################

<other wandb_sweeper_config specific params>
````

This would allow an user to easily port a wandb sweeper project into hydra without worrying about learning new parameters for this plugin.

Additionally, it would also be cool to add support to loading wandb's sweep.yaml file inside in here so that an user doesn't need to copy contents from a previously existing sweep.yaml in here. But that's low priority.